src/lib/components/Button/Button.scss

```diff
@@ -55,116 +42,137 @@ $filters: (
     contrast(100%),
 );
 
-.usa-button {
+// Note: We're setting a lot of seemingly-redundant color rules on each of
+//       these selectors for the "link displayed as a button" use-case.
+//       Removing these rules would have no impact on regular buttons but will
+//       break the styling on links using a button style.
+.usa-button,
+.usa-button.usa-link {
   width: auto;
   color: #fff;
-  background: map.get($colors, "primary");
+  background: $primary;
   .usa-icon {
     filter: map.get($filters, "white");
   }
   &:hover {
-    background: map.get($colors, "primary-dark");
+    color: #fff;
+    background: $primary-dark;
   }
   &:active {
-    background: map.get($colors, "primary-darker");
+    color: #fff;
+    background: $primary-darkest;
```

Comment:
`primary-darker` in the map is the same as the `$primary-darkest` variable (`#00284d`).